portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,12 +40,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    ArrayList<Person> messages = new ArrayList<Person>();
+    ArrayList<Pair<String, String>> messages = new ArrayList<Pair<String, String>>();
     for (Entity entity : results.asIterable()) {
       String name = (String) entity.getProperty("name");
       String text = (String) entity.getProperty("text");
 
-      Person message = new Person(name, text);
+      Pair<String, String> message = new Pair<String, String>(name, text);
```

Comment:
I personally think that the named type here, is better than the unnamed Tuple in this case. 

Person gives a lot more clear information about the purpose of the type, and also allows for easier extension later on if needed. 

What made you decide to switch to using a Pair<T, V> here?

Reply:
Grady said I should become familiar with Pairs for coding interviews so I felt this was good practice.